withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -0,0 +1,765 @@
+[*
+**  PowerOn Name:       BANNO.CHECK.WITHDRAW.V1.POW
+**  Letterfile Name:    BANNO.CHECK.WITHDRAW.V1.CFG
+**
+**  Copyright 2020 Jack Henry and Associates
+**
+**  This Banno service PowerOn allows the user to request 
+**  withdraw by check from an eligible share or loan
+**
+**  For more information check
+**  https://github.com/Banno/banno-powerons
+**
+**  Banno is not responsible for any modifications to this file
+**  made by unauthorized personnel.
+**
+**  Modification History:
```

Comment:
No need for this. We can track the modification history in github - this is redundant.